### PR TITLE
[B5] web apps: more cherry picks

### PR DESCRIPTION
**Pull request by orangejenny**
## Technical Summary
This PR is limited to changes in B5 files that aren't live in prod.

### Safety story
Does not affect production.

### Automated test coverage

no

### QA Plan

QA will come later.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/bootstrap5/form.html [hunk 1]
```diff
@@ -64,9 +64,9 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
     </form>
   </div>
   <div data-bind="visible: erroredQuestions().length > 0">
-    <div id="scroll-bottom" class="btn btn-outline-danger" style="position: fixed; bottom: 35px" title="{% trans_html_attr "Jump between required/errored questions" %}" data-bind="click: jumpToErrors">  {# todo B5: inline style #}
-      <i class='fa fa-fast-forward'> </i>
-          {% trans "Next Error" %}
+    <div id="next-error" class="btn btn-outline-danger" title="{% trans_html_attr "Jump between required/errored questions" %}" data-bind="click: jumpToErrors">  {# todo B5: inline style #}
```
Comment (biyeun) [inline corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/bootstrap5/form.html:67]: is the `todo` comment still needed?

File: corehq/apps/cloudcare/templates/cloudcare/partials/case_list/bootstrap5/tile_grouped_item.html [hunk 2]
```diff
@@ -37,12 +37,12 @@
...
                 if(resolveUri(datum)) { %>
-                  <img alt="<%- altText[index] %>" class="module-icon" style="max-width:100%; max-height:100%;" src="<%- resolveUri(datum) %>"/>  {# todo B5: inline style #}
+                  <img alt="<%- altText[index] %>" class="module-icon" src="<%- resolveUri(datum) %>"/>
                 <% } %>
               <% } else if (styles[index].displayFormat === 'ClickableIcon') {
                 if(resolveUri(datum)) { %>
                   <button class="clickable-icon">
-                    <img alt="<%- altText[index] %>" class="module-icon" src="<%- resolveUri(datum) %>" style="max-width:100%; max-height:100%;"/>  {# todo B5: inline style #}
+                    <img alt="<%- altText[index] %>" class="module-icon" src="<%- resolveUri(datum) %>" />
                     <i class="fa fa-spin fa-spinner" style="display:none"></i>  {# todo B5: inline style #}
```
Comment (biyeun) [inline corehq/apps/cloudcare/templates/cloudcare/partials/case_list/bootstrap5/tile_grouped_item.html:46]: is this `todo` comment still needed?

Comment (orangejenny) [reply corehq/apps/cloudcare/templates/cloudcare/partials/case_list/bootstrap5/tile_grouped_item.html:46]: yeah, I'm going to fix that one in a different PR

File: corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/bootstrap5/form.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/bootstrap5/form.html]: nope, fixed in c8c87cb6376

**Review by biyeun — APPROVED** [review]
just some questions on the todo comments being left behind

**Review by biyeun — APPROVED** [review]

**Pull request merged**